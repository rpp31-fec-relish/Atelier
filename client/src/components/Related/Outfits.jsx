import React from 'react';

const Outfits = (props) => {
  if (props.outfits.length === 0) {
    return '';
  } else {
    return (
      <section id="OutfitsWidget">
        <div id="OutfitItem">
          <p>
            <input type="image" alt="image"></input>
            <h6>Category</h6>
            <h6>Title</h6>
            <h6>Price</h6>
            <h6>Rating</h6>
          </p>
        </div>
        <div id="OutfitItem">
          <p>
            <input type="image" alt="image"></input>
            <h6>Category</h6>
            <h6>Title</h6>
            <h6>Price</h6>
            <h6>Rating</h6>
          </p>
        </div>
        <div id="OutfitItem">
          <p>
            <input type="image" alt="image"></input>
            <h6>Category</h6>
            <h6>Title</h6>
            <h6>Price</h6>
            <h6>Rating</h6>
          </p>
        </div>
        <div id="OutfitItem">
          <p>
            <input type="image" alt="image"></input>
            <h6>Category</h6>
            <h6>Title</h6>
            <h6>Price</h6>
            <h6>Rating</h6>
          </p>
        </div>
      </section>
    )
  }
}

export default Outfits;