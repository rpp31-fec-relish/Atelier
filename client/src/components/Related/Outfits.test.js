import Outfits from './Outfits.jsx';

import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';

Enzyme.configure({ adapter: new Adapter() });

describe('Outfits', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Outfits />);
    expect(wrapper).toMatchSnapshot();
  });
});