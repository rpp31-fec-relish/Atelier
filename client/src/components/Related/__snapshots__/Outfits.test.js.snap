// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Outfits renders correctly 1`] = `
<div
  id="outfits"
>
  <h4>
    Outfits
  </h4>
  <div
    id="outfits-product"
  >
    <input
      alt="image"
      type="image"
    />
    <h6>
      Category
    </h6>
    <h6>
      Title
    </h6>
    <h6>
      Price
    </h6>
    <h6>
      Rating
    </h6>
  </div>
</div>
`;
